awesomemotive/all-in-one-seo-pack
# Add schema BreadcrumbList

Resolves #2813

## Proposed changes

Adds breadcrumbs to the schema structured data.

## Types of changes

What types of changes does your code introduce?

- New feature (non-breaking change which adds functionality)
- Improves existing functionality

## Checklist


## Testing instructions
1. Navigate frontend and check ld+/json data in <head>
    a. Home/Main
    b. Post
    c. Child Page
    d. Taxonomy Term archive.
    e. Date archive.
    f. Author page.
    g. Search page.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
